addon/helpers/route-action.js

```diff
@@ -18,9 +26,18 @@ export default Helper.extend({
     assert('[ember-route-action-helper] Unable to lookup router', router);
 
     return function(...invocationArgs) {
+      let action;
+      let handler = emberArray(getRoutes(router)).find((route) => {
```

Comment:
Can you extract this into another shared function (`getRouteWithAction` or something)?